dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -221,110 +222,35 @@ Global Flags:
 
 有关查询结果中各参数的意义，详见[查询状态结果](/dev/reference/tools/data-migration/query-status.md#查询结果)。
```

Comment:
where is the sample result here? @csuzhangxc

Reply:
https://github.com/pingcap/docs-cn/blob/master/dev/reference/tools/data-migration/query-status.md#%E6%9F%A5%E8%AF%A2%E7%BB%93%E6%9E%9C has a complete result.